packages/core/src/managers/resource-manager.ts

```
@@ -928,291 +784,65 @@ export class ResourceManager {
-      ["PALADIN_T3_BALANCE", ResourcesIds.PaladinT3],
-      ["WHEAT_BALANCE", ResourcesIds.Wheat],
-      ["FISH_BALANCE", ResourcesIds.Fish],
-      ["LORDS_BALANCE", ResourcesIds.Lords],
-    ];
+  public static balanceWithProduction(
+    resource: ComponentValue<ClientComponents["Resource"]["schema"]>,
+    currentTick: number,
+    resourceId: ResourcesIds,
+  ): { balance: number; hasReachedMaxCapacity: boolean } {
+    const resourceWeightKg = configManager.getResourceWeightKg(resourceId);
+    const { balance, production } = this.balanceAndProduction(resource, resourceId);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)